### DISCUSSION TITLE
Call SEPIA from code

### DISCUSSION THREAD

**Discussion by daniuher** [kschan0214/sepia#39] [Q&A]
Hi! Is there a way to call Sepia from code please? I would like to specify all the algorithms and variables and run it directly without the GUI. Thank you in advance!

**Comment by kschan0214** [comment]
Hi, it is possible to run the pipeline in SEPIA without initialing the GUI. In fact, the GUI is responsible only for producing a script to run the QSM processing. 

Perhaps the easiest way to do this is to first run a full pipeline with the desired algorithms/parameters in the GUI with one of your data. After the processing, there should be a Matlab script file called "sepia_config.m" in the output directory. This file contains all the information regarding the selected pipeline and can be re-run without the SEPIA's GUI. I think you can adapt this file for your need.

**Reply by daniuher** [reply]
This is perfect, thank you so much!
